components/menu/stories/template.js

```diff
@@ -158,7 +163,8 @@ export const MenuGroup = ({
       role="presentation">
       ${heading
         ? html`<span
-            class="spectrum-Menu-sectionHeading"
+            class="spectrum-Menu-sectionHeading ${shouldTruncate ? "spectrum-Menu-itemLabel--truncate" : "" }"
+            style=${maxInlineSize ? `max-inline-size: ${maxInlineSize}px;` : ""}
```

Comment:
Could the `px` be removed and added to the values, so this is more open and could be set to other valid widths like a percentage?

Reply:
yes, I switch to not using a number input so I could make it more flexible. 
In looking at this I also caught that my changes had gotten rid of some custom styling on other stories, addressed this too.